installer/conf/kube.conf

```diff
@@ -118,6 +118,19 @@
   max_retry_wait 9m
```

Comment:
this change will have to be made in the aks yaml as well (with the 2 commands i gave you). Please track it as part of next deployment. Also, omsdocker repo needs to be updated with yaml changes + helm chart needs to be updated with yaml changes. Please track all these workitems,

Reply:
Just created work items for these.